zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -85,6 +86,7 @@
   public static Server jettyWebServer;
   public static NotebookServer notebookWsServer;
   public static Helium helium;
+  public static UsersRepo usersRepo;
```

Comment:
I don't have a more proper name, but `UsersRepo` seems a little confusing to me when it is used to stored the recent opened notes.